lib/roo_on_rails/routemaster/lifecycle_events.rb

```diff
@@ -0,0 +1,49 @@
+require 'active_support/concern'
+require 'new_relic/agent'
+
+module RooOnRails
+  module Routemaster
+    module LifecycleEvents
+      extend ActiveSupport::Concern
+
+      ROUTEMASTER_EVENT_TYPE_MAP = {
```

Comment:
I'd add a little comment — this maps _ActiveRecord_ events to Routemaster events?